docs/cluster_management.md

```diff
@@ -2,45 +2,152 @@
 
 This doc is in progress.
 
+## Overview
+
+Kubernetes is an orchestration system for managing container clusters. This document provides basic instructions for setting up and managing the cluster and assumes familiarity with Kubernetes architecture and terminology.
+
+## Prerequisites
+
+Kubernetes is written in Go and with some work it should be possible to run it anywhere where Docker can run (yes, this includes Raspberry Pi). Due to differences between Linux distributions and between cloud providers setting it up requires different amount of work on different environments. The best and simplest experience is provided by using Google Container Engine [GKE](https://cloud.google.com/container-engine/) or Google Compute Engine [GCE](https://cloud.google.com/compute/). Other environments are supported and maintained by the community, Vagrant or CoreOS on various IaaS providers being two examples.
```

Comment:
Not needed.  Covered by docs/getting-started-guide/README.md and docs/getting-started-guide/scratch.md